File: Solutions/My very 1st contest!/Rust/main.rs
// Problem: My very 1st contest!
// Link to the problem: https://www.codechef.com/problems/MY1STCONTEST
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin()
        .read_line(&mut input)
        .expect("Failed to read line");

    let mut iter = input.trim().split_whitespace();
    let n: i32 = iter
        .next()
        .expect("No value found")
        .parse()
        .expect("Failed to parse integer");
    let a: i32 = iter
        .next()
        .expect("No value found")
        .parse()
        .expect("Failed to parse integer");
    let b: i32 = iter
        .next()
        .expect("No value found")
        .parse()
        .expect("Failed to parse integer");

    println!("{} {}", n - a, n - a - b);
}

File: Solutions/My very 1st contest!/C#/main.cs
// Problem: My very 1st contest!
// Link to the problem: https://www.codechef.com/problems/MY1STCONTEST
using System;

class Program
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split();

        // Parse the input values into integers
        int n = int.Parse(input[0]);
        int a = int.Parse(input[1]);
        int b = int.Parse(input[2]);
        Console.WriteLine((n - a) + " " + (n - a - b));
    }
}
